### DISCUSSION TITLE
써클 아카이브

### DISCUSSION THREAD

**Discussion by seriousran** [Pseudo-Lab/growth-collector#4] [Announcements]
여기에 공유할만한 이야기들, 자료들, 후기들에 대한 링크를 댓글로 달아가기!

**Comment by seriousran** [comment]
나의 성장을 가로막는 벽 / 5년 후 나의 모습

**Reply by soncy0311** [reply]
성장을 가로막는 벽: 나태함...
5년 후의 나의 모습: 디지털 노마드의 삶...?ㅎㅎ

**Reply by leh60245** [reply]
잠 / 내가 만든 서비스 이용자 1,000명 달성!

**Reply by seriousran** [reply]
나의 성장을 가로막는 벽
스마트폰 (유튜브 쇼츠, 인스타 릴스) 
- 순간 순간의 시간을 뺏음 -> 집중도 하락
- 장시간의 여가/휴가 시간을 뺏음 > 시간 감소
- 눈이 피로함 -> 피로도 상승

5년후 내 모습
- 내 티셔츠를 1만장 판매할 수 있을 정도의 영향력을 가진 사람되기

**Reply by yunhee1** [reply]
성장을 가로막는 벽: 
- 더 이상 성장할 게 없다는 자만 
- 현재 상황에 대한 만족

5년 후 나의 모습: 
- 한국인 10명 중 1명 이상이 사용하는 프로덕트를 기획한 사람

**Reply by ZeVicTech** [reply]
성장을 가로막는 벽: 게으름, 고민을 오래하는 점
5년 후 나의 모습: 컴퓨터 비전에서 영향력이 있는 논문을 작성하고, 사람들에게 실질적으로 도움을 줄 수 있는 AI 서비스 개발
